core/src/main/scala/org/bitcoins/core/protocol/script/ScriptWitness.scala

```diff
@@ -106,6 +110,21 @@ sealed abstract class P2WSHWitnessV0 extends ScriptWitnessV0 {
     relevantStack.map(ECDigitalSignature.fromBytes)
   }
 
+  // Note that this is not guaranteed to work for non-standard script signatures
+  lazy val scriptSignature: ScriptSignature = {
+    val asm = stack.toVector.tail.reverse.flatMap { bytes =>
+      if (bytes.isEmpty) {
+        Vector(OP_0)
```

Comment:
Why `OP_0` if there is nothing on the stack?

Reply:
Not nothing on the stack, an empty element on the stack (a Script zero)